public/docs/dart/latest/tutorial/toh-pt6.jade

```diff
@@ -89,56 +96,108 @@ block heroes-comp-add
 block review
   //- Not showing animated gif due to differences between TS and Dart implementations.
 
-block observables-section
-  //- TBC
+block observables-section-intro
+  :marked
+    Recall that `HeroService.getHeroes()` awaits for an `http.get()`
+    response and yields a _Future_ `List<Hero>`, which is fine when we are only
+    interested in a single result.
+
+block search-criteria-intro
+  :marked
+    A [StreamController][], as its name implies, is a controller for a [Stream][] that allows us to
+    manipulate the underlying stream by adding data to it, for example.
+
+    In our sample, the underlying stream of strings (`_searchTerms.stream`) represents the hero
+    name search patterns entered by the user. Each call to `search` puts a new string into
+    the stream by calling `add` over the controller.
+
+    [Stream]: https://api.dartlang.org/stable/dart-async/Stream-class.html
+    [StreamController]: https://api.dartlang.org/stable/dart-async/StreamController-class.html
+
+block observable-transformers
+  :marked
+    Fortunately, there are stream transformers that will help us reduce the request flow.
+    We'll make fewer calls to the `HeroSearchService` and still get timely results. Here's how:
+    starting from the underlying stream of search terms we
+
+    * Debounce for 300 milliseconds, that is, wait until the flow of search terms pauses for the
```

Comment:
IMHO, the correspondence is easy to make. The problem with the Dart is that some of the commands are quite verbose, and it isn't clear how to refer to them. E.g. TS
- `debounceTime(300)`

vs. Dart
- `transform(new Debounce(new Duration(milliseconds: 300)))`

Unless you find an abbreviation like
- `transform(new Debounce(... 300)))`

acceptable. Let me know what you prefer.